apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -99,7 +88,7 @@ public static function get_instance() {
 	 */
 	public function is_supported_theme( $theme_slug ) {
 		$theme_slug = $this->normalize_theme_slug( $theme_slug );
```

Comment:
We don't need this line

Reply:
Actually I suppose the name function doesn't make sense without this. How about:

```php
//keep around temporarily for migration purposes
public function is_supported_theme( $theme_slug ) {
    return get_theme_support( 'full-site-editing' );
}

//suggested naming
public static function is_full_site_editing_active() {
    if ( apply_filters( 'a8c_disable_full_site_editing', false ) ) { 
        return false;
    }
    return get_theme_support( 'full-site-editing' );
}
```

Reply:
@kwight @mattwiebe let me know if the `apply_filters` bit makes sense

Reply:
(Almost not posting since this might just muddy the waters with other concerns, but it does also address the current issue)

I don't think that this should have the same filter name as used in [the loader](https://github.com/Automattic/wp-calypso/blob/5cde6adfda0f6e3754b3a6e223d887365b0b6045/apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php#L36-L38). It mixes concerns (loading on the one hand, marking active on another). We have to be able to load the plugin to even run this function, presently.

Also, since the plugin is already namespaced, I think we could just go with `is_active` for the function name. Something like 

```php
//keep around temporarily for migration purposes
public function is_supported_theme( $theme_slug ) { //in reality is_current_theme_supported
    return current_theme_supports( 'full-site-editing' );
}

//suggested naming
public static function is_active() {
    if ( ! apply_filters( 'a8c_is_full_site_editing_active', true ) ) { 
        return false;
    }
    return current_theme_supports( 'full-site-editing' );
}
// called with:
\A8C\FSE\Full_Site_Editing::is_active();
```

And really, at that point, we probably want to ensure that at least the `\A8C\FSE\Full_Site_Editing` class is always loaded. Because otherwise we have this function that we can't even call. Or maybe we just put a version of it in the base plugin file. Hmm.

Reply:
We can do the rest of the tidying across multiple PRs. For example this one just updates is_supported_theme and maybe adds the better is_active check that no one calls yet. (I'd be okay on a different PR to add that). 

Just noting that naming is completely off atm :D But I think callers should be okay.